core/scss/utilities/mixins/typography/_font-splash.scss

```diff
@@ -8,15 +8,10 @@
 // Style guide: Mixins.Typography.font-splash
 //
 @mixin font-splash {
-  @include h3;
+  @include modular-typography(4);
   @include margin(null null 0);
 
-  @media #{$breakpoint-sm} {
-    @include h1;
-  }
-
   @media #{$breakpoint-md} {
-    font-size: $splash-font-size;
     font-weight: $font-bold;
```

Comment:
```suggestion
    font-weight: $font-x-bold;
```

Reply:
Just making it bold (regular definition as font-weight: 700) instead of semibold.

Reply:
No longer needed since variables were updated accordingly.